_documentation/user/flags/jamming.md

```diff
@@ -5,4 +5,19 @@ type: Bad
 summary: Radar doesn't work. Can still see.
 ---
 
-...
+The Jamming flag causes static to block out radar, outside window viewing perspective is unaffected.
+
+## Tactics
+
+Jamming prevents any effective usage of radar, as it fills the radar with static.
+
+Countering the effects of Jamming requires good knowledge of the map and layout, along with effective moment patterns and positioning.
```

Comment:
> "along with effective moment patterns and positioning."

What?

Reply:
The effect is less about the map layout and more about suddenly not knowing who is around you, which can both affect you and your teammates as you may not notice an enemy or a teammate nearby.

Reply:
> What?

I usually try to have a side to side pattern in movement, so it can cover around 180 degrees in total for viewing, and usually try to be where visibility is high.

> The effect is less about the map layout and more about suddenly not knowing who is around you, which can both affect you and your teammates as you may not notice an enemy or a teammate nearby.

Will revise it to reflect this.